src/js/components/covid19/awardSpendingAgency/AwardSpendingAgency.jsx

```diff
@@ -103,30 +98,9 @@ const AwardSpendingAgency = () => {
             } else {
                 params.filter.award_type_codes = awardTypeGroups[activeTab.internal];
             }
-
-            fetchAwardAmounts(params).promise
```

Comment:
These calls are now being made in `SummaryInsightsContainer`

Reply:
Ah this is nice. I also did something like this in my loading state PR.